bears/coffee_script/CoffeeLintBear.py

```diff
@@ -22,11 +23,285 @@ class CoffeeLintBear:
+            f = "#{bar}"
+            ```
+            is correct.
+        :param no_stand_alone_at:
+            Prohibits the use of stand alone  ``@``.
+
+            Example: If ``no_stand_alone_at = True``
+            ```
+            @ notok
+            not(@).ok
+            @::
+            ```
+            are prohibited, whereas
+            ```
+            @alright
+            @(fn)
+            @ok()
+            @[ok]
+            @ok()
+            ```
+            are accepted.
+        :param disable_throwing_strings:
+            Disables throwing string literals or interpolation. It is forbidden
+            by default.
+
+            Example: If ``disable_throwing_strings = True``
+            ```
+            throw 'my error'
+            throw "#{1234}"
+            ```
+            will not be permitted.
+        :param allow_trailing_semicolons:
+            Prohibits trailing semicolons when ``False`` since they are
+            not useful. The semicolon is meaningful as there's another
+            instruction on the same line.
+
+            Example: If ``allow_trailing_semicolon = False``
+            ```
+            x = '1234'; console.log(x)
+            ```
+            here the semicolon is meaningful.
+            ```
+            alert('end of line');
+            ```
+            the semicolon is redundant.
+        :param allow_trailing_whitespaces:
+            Checks whether they are trailing whitespace in the code or not.
+        :param no_trailing_whitespaces_affects_comments:
+            Determines if ``no_trailing_whitespaces`` rule should be applied to
+            comments.
+        :param no_trailing_whitespaces_affects_empty_lines:
+            Determines if ``no_trailing_whitespace`` rule should be applied to
+            empty lines.
+        :param no_unnecessary_double_quotes:
+            Prohibits double quotes unless string interpolation is used or the
+            strings contain single quotes.
+        :param use_english_operator:
+            Determines if ``and``, ``or``, ``is`` and ``isnt`` should be used
+            instead of ``&&``, ``||``, ``==`` and ``!=``.
+        :param spaces_around_operators:
+            Enforces that operators have spaces around them.
+        :param space_after_comma:
+            Checks if there is a blank space after commas.
+        :param cyclomatic_complexity:
+            Cyclomatic complexity of the file.
+        :param prevent_duplicate_keys:
+            Prevents defining duplicate keys in object literals and classes.
+        :param enforce_parentheses_on_non_empty_constructors:
+            Requires constructors with parameters to include parentheses.
+        :param consistent_line_endings:
+            Ensures that the file uses only ``windows`` or ``unix`` line
+            endings.
+        :param consistent_line_endings_style:
+            The option to ``line_endings``, its value is either ``unix`` or
+            ``windows``.
```

Comment:
ah... please remove `consistent_line_endings` and use `None` as default or introduce some special value that means `don't care`